doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -51,4 +51,151 @@ $ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[
 
 @endcode
 
+
+Running pre-trained YOLO model in OpenCV
+----------------------------------------
+
+Deploying pre-trained models is a common task in machine learning, particularly when working with hardware that does not support certain frameworks like PyTorch. This guide provides a comprehensive overview of exporting pre-trained YOLO family models from PyTorch and deploying them using OpenCV's runtime framework. For demonstration purposes, we will focus on the [YOLOX](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/LICENSE) model, but the methodology applies to other supported<sup>*</sup> models.
+
+<sup>*</sup> Currently, OpenCV supports the following YOLO models: YOLOX, YOLOv8, YOLOv7, YOLOv6, YOLOv5, and YOLOv4. This support includes pre and post-processing routines specific to these models. While other models can be used with OpenCV, they may require custom implementation of these routines.
```

Comment:
Yolo NAS. Also need to say that older Yolos with Darkenet are supported too, but are out of scope for the tutorials.